.github/workflows/tests.yml

```
@@ -0,0 +1,42 @@
+      - master
+
+jobs:
+  tests:
+    name: Unit Tests [PHP ${{ matrix.php-versions }}]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['8.3', '8.4']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)